bindings/ocaml/AutoConfig2.ml

```diff
@@ -17,8 +17,8 @@ let has_feature = function
   | SHAEXT -> everCrypt_AutoConfig2_has_shaext ()
   | AES_NI -> everCrypt_AutoConfig2_has_aesni ()
   | PCLMULQDQ -> everCrypt_AutoConfig2_has_pclmulqdq ()
-  | AVX -> everCrypt_AutoConfig2_has_avx ()
-  | AVX2 -> everCrypt_AutoConfig2_has_avx2 ()
+  | VEC128 -> everCrypt_AutoConfig2_has_vec128 ()
```

Comment:
I am a bit skeptical about this renaming but ok, generally I agree with you that this shouldn't be architecture specific...